3.37.4 is failing native build

### Describe the bug

https://github.com/quarkusio/quarkus/releases/tag/3.37.4

```
Build resources:
 - 23.29GB of memory (75.6% of 30.82GB system memory, determined at start)
 - 8 thread(s) (100.0% of 8 available processor(s), determined at start)
[2/8] Performing analysis...  []                                                                        (28.3s @ 1.17GB)
   13,882 reachable types   (82.0% of   16,926 total)
   18,963 reachable fields  (56.2% of   33,721 total)
   68,241 reachable methods (50.8% of  134,293 total)
    4,780 types, 1,977 fields, and 15,139 methods registered for reflection
Fatal error: com.oracle.graal.pointsto.util.AnalysisError$ParsingError: Error encountered while parsing io.netty.handler.ssl.SslContext.newClientContextInternal(SslContext.java:844) 
Parsing context:
   at io.netty.handler.ssl.SslContextBuilder.build(SslContextBuilder.java:648)
   at io.grpc.netty.ProtocolNegotiators.from(ProtocolNegotiators.java:146)
   at io.grpc.netty.NettyChannelProvider.newChannelBuilder(NettyChannelProvider.java:50)
   at root method.(Unknown Source)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AnalysisError.parsingError(AnalysisError.java:149)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:184)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.ensureFlowsGraphCreated(MethodTypeFlow.java:153)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.getOrCreateMethodFlowsGraphInfo(MethodTypeFlow.java:111)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.typestate.DefaultStaticInvokeTypeFlow.lambda$update$0(DefaultStaticInvokeTypeFlow.java:75)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.LightImmutableCollection.forEach(LightImmutableCollection.java:90)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.typestate.DefaultStaticInvokeTypeFlow.update(DefaultStaticInvokeTypeFlow.java:74)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.PointsToAnalysis$1.run(PointsToAnalysis.java:491)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.executeCommand(CompletionExecutor.java:187)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.lambda$executeService$0(CompletionExecutor.java:171)
	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1423)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188)
Caused by: org.graalvm.compiler.debug.GraalError: com.oracle.svm.core.util.UserError$UserException: Class initialization of io.netty.internal.tcnative.SSLPrivateKeyMethod failed. Use the option 
    '--initialize-at-run-time=io.netty.internal.tcnative.SSLPrivateKeyMethod'
 to explicitly request initialization of this class at run time.
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AnalysisFuture.setException(AnalysisFuture.java:49)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:322)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AnalysisFuture.ensureDone(AnalysisFuture.java:63)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.ensureOnTypeReachableTaskDone(AnalysisType.java:696)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.onReachable(AnalysisType.java:590)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AtomicUtils.atomicSetAndRun(AtomicUtils.java:49)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.lambda$registerAsReachable$8(AnalysisType.java:562)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.forAllSuperTypes(AnalysisType.java:676)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.forAllSuperTypes(AnalysisType.java:679)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.forAllSuperTypes(AnalysisType.java:659)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.forAllSuperTypes(AnalysisType.java:655)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.registerAsReachable(AnalysisType.java:562)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.lookupAllowUnresolved(AnalysisUniverse.java:368)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.lookup(AnalysisUniverse.java:337)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.convertFields(AnalysisType.java:1163)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.initializeInstanceFields(AnalysisType.java:1150)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.getInstanceFields(AnalysisType.java:1141)
	at jdk.internal.vm.compiler/org.graalvm.compiler.nodes.virtual.VirtualInstanceNode.<init>(VirtualInstanceNode.java:47)
	at jdk.internal.vm.compiler/org.graalvm.compiler.nodes.java.NewInstanceNode.virtualize(NewInstanceNode.java:85)
	at jdk.internal.vm.compiler/org.graalvm.compiler.virtual.phases.ea.PartialEscapeClosure.virtualize(PartialEscapeClosure.java:317)
	at jdk.internal.vm.compiler/org.graalvm.compiler.virtual.phases.ea.PartialEscapeClosure.processVirtualizable(PartialEscapeClosure.java:313)
	at jdk.internal.vm.compiler/org.graalvm.compiler.virtual.phases.ea.PartialEscapeClosure.processNodeInternal(PartialEscapeClosure.java:228)
	at jdk.internal.vm.compiler/org.graalvm.compiler.virtual.phases.ea.PartialEscapeClosure.processNode(PartialEscapeClosure.java:211)
	at jdk.internal.vm.compiler/org.graalvm.compiler.virtual.phases.ea.PartialEscapeClosure$Final.processNode(PartialEscapeClosure.java:171)
	at jdk.internal.vm.compiler/org.graalvm.compiler.virtual.phases.ea.EffectsClosure.processBlock(EffectsClosure.java:277)
	at jdk.internal.vm.compiler/org.graalvm.compiler.virtual.phases.ea.EffectsClosure.processBlock(EffectsClosure.java:72)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.graph.ReentrantBlockIterator.apply(ReentrantBlockIterator.java:150)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.graph.ReentrantBlockIterator.apply(ReentrantBlockIterator.java:126)
	at jdk.internal.vm.compiler/org.graalvm.compiler.virtual.phases.ea.EffectsPhase.runAnalysis(EffectsPhase.java:114)
	at jdk.internal.vm.compiler/org.graalvm.compiler.virtual.phases.ea.PartialEscapePhase.run(PartialEscapePhase.java:141)
	at jdk.internal.vm.compiler/org.graalvm.compiler.virtual.phases.ea.PartialEscapePhase.run(PartialEscapePhase.java:80)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.BasePhase.apply(BasePhase.java:434)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.BasePhase.apply(BasePhase.java:322)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.optimizeAfterParsing(SVMHost.java:596)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.methodAfterParsingHook(SVMHost.java:567)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.AnalysisParsedGraph.optimizeAndEncode(AnalysisParsedGraph.java:165)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.AnalysisParsedGraph.parseBytecode(AnalysisParsedGraph.java:154)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.parseGraph(AnalysisMethod.java:895)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.ensureGraphParsedHelper(AnalysisMethod.java:860)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.ensureGraphParsed(AnalysisMethod.java:843)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.phases.InlineBeforeAnalysisGraphDecoder.lookupEncodedGraph(InlineBeforeAnalysisGraphDecoder.java:198)
	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.doInline(PEGraphDecoder.java:1211)
	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.tryInline(PEGraphDecoder.java:1194)
	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.trySimplifyInvoke(PEGraphDecoder.java:1049)
	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.handleInvokeWithCallTarget(PEGraphDecoder.java:1001)
	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.handleInvoke(PEGraphDecoder.java:987)
	at jdk.internal.vm.compiler/org.graalvm.compiler.nodes.GraphDecoder.processNextNode(GraphDecoder.java:921)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.phases.InlineBeforeAnalysisGraphDecoder.processNextNode(InlineBeforeAnalysisGraphDecoder.java:378)
	at jdk.internal.vm.compiler/org.graalvm.compiler.nodes.GraphDecoder.decode(GraphDecoder.java:650)
	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.decode(PEGraphDecoder.java:892)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.phases.InlineBeforeAnalysis.decodeGraph(InlineBeforeAnalysis.java:76)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.parse(MethodTypeFlowBuilder.java:195)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.apply(MethodTypeFlowBuilder.java:621)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:167)
	... 14 more
------------------------------------------------------------------------------------------------------------------------
Caused by: com.oracle.svm.core.util.UserError$UserException: Class initialization of io.netty.internal.tcnative.SSLPrivateKeyMethod failed. Use the option 
    '--initialize-at-run-time=io.netty.internal.tcnative.SSLPrivateKeyMethod'
 to explicitly request initialization of this class at run time.
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.util.UserError.abort(UserError.java:85)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.ClassInitializationSupport.ensureClassInitialized(ClassInitializationSupport.java:195)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.AllowAllHostedUsagesClassInitializationSupport.computeInitKindAndMaybeInitializeClass(AllowAllHostedUsagesClassInitializationSupport.java:191)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.AllowAllHostedUsagesClassInitializationSupport.computeInitKindAndMaybeInitializeClass(AllowAllHostedUsagesClassInitializationSupport.java:129)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.ClassInitializationSupport.maybeInitializeAtBuildTime(ClassInitializationSupport.java:161)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.ClassInitializationSupport.maybeInitializeAtBuildTime(ClassInitializationSupport.java:150)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.onTypeReachable(SVMHost.java:310)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.onTypeReachable(AnalysisUniverse.java:699)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.lambda$new$0(AnalysisType.java:310)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	... 66 more
Caused by: java.lang.UnsatisfiedLinkError: 'int io.netty.internal.tcnative.NativeStaticallyReferencedJniMethods.sslSignRsaPkcsSha1()'
	at io.netty.internal.tcnative.NativeStaticallyReferencedJniMethods.sslSignRsaPkcsSha1(Native Method)
	at io.netty.internal.tcnative.SSLPrivateKeyMethod.<clinit>(SSLPrivateKeyMethod.java:22)
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized0(Native Method)
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized(Unsafe.java:1160)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.ClassInitializationSupport.ensureClassInitialized(ClassInitializationSupport.java:177)
	... 75 more
                        3.4s (8.9% of total time) in 209 GCs | Peak RSS: 1.83GB | CPU load: 5.42
========================================================================================================================
Finished generating 'passenger-age-match-api-2.0.0-runner' in 36.9s.
168106 [INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildRunner] docker cp 41452f8cb2469a806d13954114fcabf9cb1526cdee97c89a9b098b345223eafd:/project/passenger-age-match-api-2.0.0-runner /builds/csitp/csp/passenger-age-match/lambda-service/target/passenger-age-match-api-2.0.0-native-image-source-jar
168131 [ERROR] [io.quarkus.deployment.pkg.steps.NativeImageBuildRunner] Failed to copy native image from container volume back to the host.
io.smallrye.common.process.AbnormalExitException: Process exited abnormally (pid 234) with exit code 1 with error output:
 > Error response from daemon: Could not find the file /project/passenger-age-match-api-2.0.0-runner in container 41452f8cb2469a806d13954114fcabf9cb1526cdee97c89a9b098b345223eafd
    at io.smallrye.common.process.PipelineRunner.collectProblems (PipelineRunner.java:559)
    at io.smallrye.common.process.ProcessRunner.complete (ProcessRunner.java:165)
    at io.smallrye.common.process.ProcessRunner.run (ProcessRunner.java:95)
    at io.smallrye.common.process.ProcessBuilderImpl.run (ProcessBuilderImpl.java:199)
    at io.quarkus.deployment.pkg.steps.NativeImageBuildRunner.runCommand (NativeImageBuildRunner.java:133)
    at io.quarkus.deployment.pkg.steps.NativeImageBuildRunner.runCommand (NativeImageBuildRunner.java:150)
    at io.quarkus.deployment.pkg.steps.NativeImageBuildRemoteContainerRunner.copyFromContainerVolume (NativeImageBuildRemoteContainerRunner.java:121)
    at io.quarkus.deployment.pkg.steps.NativeImageBuildRemoteContainerRunner.postBuild (NativeImageBuildRemoteContainerRunner.java:65)
    at io.quarkus.deployment.pkg.steps.NativeImageBuildRunner.build (NativeImageBuildRunner.java:94)
    at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.build (NativeImageBuildStep.java:300)
    at java.lang.invoke.MethodHandle.invokeWithArguments (MethodHandle.java:733)
    at io.quarkus.deployment.ExtensionLoader$3.execute (ExtensionLoader.java:811)
    at io.quarkus.builder.BuildContext.run (BuildContext.java:242)
    at org.jboss.threads.ContextHandler$1.runWith (ContextHandler.java:18)
    at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith (EnhancedQueueExecutor.java:2651)
    at org.jboss.threads.EnhancedQueueExecutor$Task.run (EnhancedQueueExecutor.java:2630)
    at org.jboss.threads.EnhancedQueueExecutor.runThreadBody (EnhancedQueueExecutor.java:1622)
    at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run (EnhancedQueueExecutor.java:1589)
    at java.lang.Thread.run (Thread.java:1583)
    at org.jboss.threads.JBossThread.run (JBossThread.java:501)
168132 [INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildRunner] docker cp 41452f8cb2469a806d13954114fcabf9cb1526cdee97c89a9b098b345223eafd:/project/build-artifacts.json /builds/csitp/csp/passenger-age-match/lambda-service/target/passenger-age-match-api-2.0.0-native-image-source-jar
168155 [INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildRunner] docker container rm 41452f8cb2469a806d13954114fcabf9cb1526cdee97c89a9b098b345223eafd
168180 [INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildRunner] docker volume rm quarkus-native-builder-image-project-volume
168222 [INFO] ------------------------------------------------------------------------
168222 [INFO] BUILD FAILURE
168222 [INFO] ------------------------------------------------------------------------
168224 [INFO] Total time:  02:47 min
168225 [INFO] Finished at: 2026-07-23T14:34:15Z
168225 [INFO] ------------------------------------------------------------------------
168227 [ERROR] Failed to execute goal io.quarkus.platform:quarkus-maven-plugin:3.37.4:build (default) on project passenger-age-match-api: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
168227 [ERROR] 	[error]: Build step io.quarkus.deployment.pkg.steps.NativeImageBuildStep#build threw an exception: io.quarkus.deployment.pkg.steps.NativeImageBuildStep$ImageGenerationFailureException: Image generation failed
168227 [ERROR] 	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.imageGenerationFailed(NativeImageBuildStep.java:616)
168227 [ERROR] 	at 
```

### Expected behavior

native build should work after netty upgrade to 4.1.136

### Actual behavior

native build is  failing after netty upgrade to 4.1.136

```
Build resources:
 - 23.29GB of memory (75.6% of 30.82GB system memory, determined at start)
 - 8 thread(s) (100.0% of 8 available processor(s), determined at start)
[2/8] Performing analysis...  []                                                                        (28.3s @ 1.17GB)
   13,882 reachable types   (82.0% of   16,926 total)
   18,963 reachable fields  (56.2% of   33,721 total)
   68,241 reachable methods (50.8% of  134,293 total)
    4,780 types, 1,977 fields, and 15,139 methods registered for reflection
Fatal error: com.oracle.graal.pointsto.util.AnalysisError$ParsingError: Error encountered while parsing io.netty.handler.ssl.SslContext.newClientContextInternal(SslContext.java:844) 
Parsing context:
   at io.netty.handler.ssl.SslContextBuilder.build(SslContextBuilder.java:648)
   at io.grpc.netty.ProtocolNegotiators.from(ProtocolNegotiators.java:146)
   at io.grpc.netty.NettyChannelProvider.newChannelBuilder(NettyChannelProvider.java:50)
   at root method.(Unknown Source)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AnalysisError.parsingError(AnalysisError.java:149)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:184)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.ensureFlowsGraphCreated(MethodTypeFlow.java:153)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.getOrCreateMethodFlowsGraphInfo(MethodTypeFlow.java:111)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.typestate.DefaultStaticInvokeTypeFlow.lambda$update$0(DefaultStaticInvokeTypeFlow.java:75)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.LightImmutableCollection.forEach(LightImmutableCollection.java:90)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.typestate.DefaultStaticInvokeTypeFlow.update(DefaultStaticInvokeTypeFlow.java:74)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.PointsToAnalysis$1.run(PointsToAnalysis.java:491)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.executeCommand(CompletionExecutor.java:187)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.lambda$executeService$0(CompletionExecutor.java:171)
	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1423)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188)
Caused by: org.graalvm.compiler.debug.GraalError: com.oracle.svm.core.util.UserError$UserException: Class initialization of io.netty.internal.tcnative.SSLPrivateKeyMethod failed. Use the option 
    '--initialize-at-run-time=io.netty.internal.tcnative.SSLPrivateKeyMethod'
 to explicitly request initialization of this class at run time.
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AnalysisFuture.setException(AnalysisFuture.java:49)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:322)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AnalysisFuture.ensureDone(AnalysisFuture.java:63)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.ensureOnTypeReachableTaskDone(AnalysisType.java:696)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.onReachable(AnalysisType.java:590)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AtomicUtils.atomicSetAndRun(AtomicUtils.java:49)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.lambda$registerAsReachable$8(AnalysisType.java:562)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.forAllSuperTypes(AnalysisType.java:676)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.forAllSuperTypes(AnalysisType.java:679)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.forAllSuperTypes(AnalysisType.java:659)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.forAllSuperTypes(AnalysisType.java:655)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.registerAsReachable(AnalysisType.java:562)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.lookupAllowUnresolved(AnalysisUniverse.java:368)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.lookup(AnalysisUniverse.java:337)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.convertFields(AnalysisType.java:1163)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.initializeInstanceFields(AnalysisType.java:1150)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.getInstanceFields(AnalysisType.java:1141)
	at jdk.internal.vm.compiler/org.graalvm.compiler.nodes.virtual.VirtualInstanceNode.<init>(VirtualInstanceNode.java:47)
	at jdk.internal.vm.compiler/org.graalvm.compiler.nodes.java.NewInstanceNode.virtualize(NewInstanceNode.java:85)
	at jdk.internal.vm.compiler/org.graalvm.compiler.virtual.phases.ea.PartialEscapeClosure.virtualize(PartialEscapeClosure.java:317)
	at jdk.internal.vm.compiler/org.graalvm.compiler.virtual.phases.ea.PartialEscapeClosure.processVirtualizable(PartialEscapeClosure.java:313)
	at jdk.internal.vm.compiler/org.graalvm.compiler.virtual.phases.ea.PartialEscapeClosure.processNodeInternal(PartialEscapeClosure.java:228)
	at jdk.internal.vm.compiler/org.graalvm.compiler.virtual.phases.ea.PartialEscapeClosure.processNode(PartialEscapeClosure.java:211)
	at jdk.internal.vm.compiler/org.graalvm.compiler.virtual.phases.ea.PartialEscapeClosure$Final.processNode(PartialEscapeClosure.java:171)
	at jdk.internal.vm.compiler/org.graalvm.compiler.virtual.phases.ea.EffectsClosure.processBlock(EffectsClosure.java:277)
	at jdk.internal.vm.compiler/org.graalvm.compiler.virtual.phases.ea.EffectsClosure.processBlock(EffectsClosure.java:72)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.graph.ReentrantBlockIterator.apply(ReentrantBlockIterator.java:150)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.graph.ReentrantBlockIterator.apply(ReentrantBlockIterator.java:126)
	at jdk.internal.vm.compiler/org.graalvm.compiler.virtual.phases.ea.EffectsPhase.runAnalysis(EffectsPhase.java:114)
	at jdk.internal.vm.compiler/org.graalvm.compiler.virtual.phases.ea.PartialEscapePhase.run(PartialEscapePhase.java:141)
	at jdk.internal.vm.compiler/org.graalvm.compiler.virtual.phases.ea.PartialEscapePhase.run(PartialEscapePhase.java:80)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.BasePhase.apply(BasePhase.java:434)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.BasePhase.apply(BasePhase.java:322)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.optimizeAfterParsing(SVMHost.java:596)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.methodAfterParsingHook(SVMHost.java:567)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.AnalysisParsedGraph.optimizeAndEncode(AnalysisParsedGraph.java:165)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.AnalysisParsedGraph.parseBytecode(AnalysisParsedGraph.java:154)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.parseGraph(AnalysisMethod.java:895)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.ensureGraphParsedHelper(AnalysisMethod.java:860)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.ensureGraphParsed(AnalysisMethod.java:843)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.phases.InlineBeforeAnalysisGraphDecoder.lookupEncodedGraph(InlineBeforeAnalysisGraphDecoder.java:198)
	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.doInline(PEGraphDecoder.java:1211)
	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.tryInline(PEGraphDecoder.java:1194)
	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.trySimplifyInvoke(PEGraphDecoder.java:1049)
	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.handleInvokeWithCallTarget(PEGraphDecoder.java:1001)
	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.handleInvoke(PEGraphDecoder.java:987)
	at jdk.internal.vm.compiler/org.graalvm.compiler.nodes.GraphDecoder.processNextNode(GraphDecoder.java:921)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.phases.InlineBeforeAnalysisGraphDecoder.processNextNode(InlineBeforeAnalysisGraphDecoder.java:378)
	at jdk.internal.vm.compiler/org.graalvm.compiler.nodes.GraphDecoder.decode(GraphDecoder.java:650)
	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.decode(PEGraphDecoder.java:892)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.phases.InlineBeforeAnalysis.decodeGraph(InlineBeforeAnalysis.java:76)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.parse(MethodTypeFlowBuilder.java:195)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.apply(MethodTypeFlowBuilder.java:621)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:167)
	... 14 more
------------------------------------------------------------------------------------------------------------------------
Caused by: com.oracle.svm.core.util.UserError$UserException: Class initialization of io.netty.internal.tcnative.SSLPrivateKeyMethod failed. Use the option 
    '--initialize-at-run-time=io.netty.internal.tcnative.SSLPrivateKeyMethod'
 to explicitly request initialization of this class at run time.
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.util.UserError.abort(UserError.java:85)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.ClassInitializationSupport.ensureClassInitialized(ClassInitializationSupport.java:195)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.AllowAllHostedUsagesClassInitializationSupport.computeInitKindAndMaybeInitializeClass(AllowAllHostedUsagesClassInitializationSupport.java:191)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.AllowAllHostedUsagesClassInitializationSupport.computeInitKindAndMaybeInitializeClass(AllowAllHostedUsagesClassInitializationSupport.java:129)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.ClassInitializationSupport.maybeInitializeAtBuildTime(ClassInitializationSupport.java:161)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.ClassInitializationSupport.maybeInitializeAtBuildTime(ClassInitializationSupport.java:150)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.onTypeReachable(SVMHost.java:310)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.onTypeReachable(AnalysisUniverse.java:699)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisType.lambda$new$0(AnalysisType.java:310)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	... 66 more
Caused by: java.lang.UnsatisfiedLinkError: 'int io.netty.internal.tcnative.NativeStaticallyReferencedJniMethods.sslSignRsaPkcsSha1()'
	at io.netty.internal.tcnative.NativeStaticallyReferencedJniMethods.sslSignRsaPkcsSha1(Native Method)
	at io.netty.internal.tcnative.SSLPrivateKeyMethod.<clinit>(SSLPrivateKeyMethod.java:22)
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized0(Native Method)
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized(Unsafe.java:1160)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.ClassInitializationSupport.ensureClassInitialized(ClassInitializationSupport.java:177)
	... 75 more
                        3.4s (8.9% of total time) in 209 GCs | Peak RSS: 1.83GB | CPU load: 5.42
========================================================================================================================
Finished generating 'passenger-age-match-api-2.0.0-runner' in 36.9s.
168106 [INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildRunner] docker cp 41452f8cb2469a806d13954114fcabf9cb1526cdee97c89a9b098b345223eafd:/project/passenger-age-match-api-2.0.0-runner /builds/csitp/csp/passenger-age-match/lambda-service/target/passenger-age-match-api-2.0.0-native-image-source-jar
168131 [ERROR] [io.quarkus.deployment.pkg.steps.NativeImageBuildRunner] Failed to copy native image from container volume back to the host.
io.smallrye.common.process.AbnormalExitException: Process exited abnormally (pid 234) with exit code 1 with error output:
 > Error response from daemon: Could not find the file /project/passenger-age-match-api-2.0.0-runner in container 41452f8cb2469a806d13954114fcabf9cb1526cdee97c89a9b098b345223eafd
    at io.smallrye.common.process.PipelineRunner.collectProblems (PipelineRunner.java:559)
    at io.smallrye.common.process.ProcessRunner.complete (ProcessRunner.java:165)
    at io.smallrye.common.process.ProcessRunner.run (ProcessRunner.java:95)
    at io.smallrye.common.process.ProcessBuilderImpl.run (ProcessBuilderImpl.java:199)
    at io.quarkus.deployment.pkg.steps.NativeImageBuildRunner.runCommand (NativeImageBuildRunner.java:133)
    at io.quarkus.deployment.pkg.steps.NativeImageBuildRunner.runCommand (NativeImageBuildRunner.java:150)
    at io.quarkus.deployment.pkg.steps.NativeImageBuildRemoteContainerRunner.copyFromContainerVolume (NativeImageBuildRemoteContainerRunner.java:121)
    at io.quarkus.deployment.pkg.steps.NativeImageBuildRemoteContainerRunner.postBuild (NativeImageBuildRemoteContainerRunner.java:65)
    at io.quarkus.deployment.pkg.steps.NativeImageBuildRunner.build (NativeImageBuildRunner.java:94)
    at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.build (NativeImageBuildStep.java:300)
    at java.lang.invoke.MethodHandle.invokeWithArguments (MethodHandle.java:733)
    at io.quarkus.deployment.ExtensionLoader$3.execute (ExtensionLoader.java:811)
    at io.quarkus.builder.BuildContext.run (BuildContext.java:242)
    at org.jboss.threads.ContextHandler$1.runWith (ContextHandler.java:18)
    at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith (EnhancedQueueExecutor.java:2651)
    at org.jboss.threads.EnhancedQueueExecutor$Task.run (EnhancedQueueExecutor.java:2630)
    at org.jboss.threads.EnhancedQueueExecutor.runThreadBody (EnhancedQueueExecutor.java:1622)
    at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run (EnhancedQueueExecutor.java:1589)
    at java.lang.Thread.run (Thread.java:1583)
    at org.jboss.threads.JBossThread.run (JBossThread.java:501)
168132 [INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildRunner] docker cp 41452f8cb2469a806d13954114fcabf9cb1526cdee97c89a9b098b345223eafd:/project/build-artifacts.json /builds/csitp/csp/passenger-age-match/lambda-service/target/passenger-age-match-api-2.0.0-native-image-source-jar
168155 [INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildRunner] docker container rm 41452f8cb2469a806d13954114fcabf9cb1526cdee97c89a9b098b345223eafd
168180 [INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildRunner] docker volume rm quarkus-native-builder-image-project-volume
168222 [INFO] ------------------------------------------------------------------------
168222 [INFO] BUILD FAILURE
168222 [INFO] ------------------------------------------------------------------------
168224 [INFO] Total time:  02:47 min
168225 [INFO] Finished at: 2026-07-23T14:34:15Z
168225 [INFO] ------------------------------------------------------------------------
168227 [ERROR] Failed to execute goal io.quarkus.platform:quarkus-maven-plugin:3.37.4:build (default) on project passenger-age-match-api: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
168227 [ERROR] 	[error]: Build step io.quarkus.deployment.pkg.steps.NativeImageBuildStep#build threw an exception: io.quarkus.deployment.pkg.steps.NativeImageBuildStep$ImageGenerationFailureException: Image generation failed
168227 [ERROR] 	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.imageGenerationFailed(NativeImageBuildStep.java:616)
168227 [ERROR] 	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.build(NativeImageBuildStep.java:306)
168227 [ERROR] 	at java.base/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:733)
168227 [ERROR] 	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:811)
168227 [ERROR] 	at io.quarkus.builder.BuildContext.run(BuildContext.java:242)
168227 [ERROR] 	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
168227 [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2651)
168227 [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2630)
168227 [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor.runThreadBody(EnhancedQueueExecutor.java:1622)
168227 [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1589)
168227 [ERROR] 	at java.base/java.lang.Thread.run(Thread.java:1583)
168227 [ERROR] 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
168227 [ERROR] Caused by: io.smallrye.common.process.AbnormalExitException: Process exited abnormally (pid 219) with exit code 1 with error output:
168227 [ERROR]  > Fatal error: com.oracle.graal.pointsto.util.AnalysisError$ParsingError: Error encountered while parsing io.netty.handler.ssl.SslContext.newClientContextInternal(SslContext.java:844) 
168227 [ERROR]  > Parsing context:
168227 [ERROR]  >    at io.netty.handler.ssl.SslContextBuilder.build(SslContextBuilder.java:648)
168227 [ERROR]  >    at io.grpc.netty.ProtocolNegotiators.from(ProtocolNegotiators.java:146)
168227 [ERROR]  >    at io.grpc.netty.NettyChannelProvider.newChannelBuilder(NettyChannelProvider.java:50)
168227 [ERROR]  > … (skipped 97 line(s)) …
168227 [ERROR]  > 	at io.netty.internal.tcnative.SSLPrivateKeyMethod.<clinit>(SSLPrivateKeyMethod.java:22)
168227 [ERROR]  > 	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized0(Native Method)
168227 [ERROR]  > 	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized(Unsafe.java:1160)
168227 [ERROR]  > 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.ClassInitializationSupport.ensureClassInitialized(ClassInitializationSupport.java:177)
168228 [ERROR]  > 	... 75 more
168228 [ERROR] 	at io.smallrye.common.process.PipelineRunner.collectProblems(PipelineRunner.java:559)
168228 [ERROR] 	at io.smallrye.common.process.ProcessRunner.complete(ProcessRunner.java:165)
168228 [ERROR] 	at io.smallrye.common.process.ProcessRunner.run(ProcessRunner.java:95)
168228 [ERROR] 	at io.smallrye.common.process.ProcessBuilderImpl.run(ProcessBuilderImpl.java:199)
168228 [ERROR] 	at io.quarkus.deployment.pkg.steps.NativeImageBuildRunner.build(NativeImageBuildRunner.java:77)
168228 [ERROR] 	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.build(NativeImageBuildStep.java:300)
168228 [ERROR] 	... 10 more
168228 [ERROR] -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.quarkus.platform:quarkus-maven-plugin:3.37.4:build (default) on project passenger-age-match-api: Failed to build quarkus application
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:375)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:351)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:171)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:163)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:294)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:960)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
    at java.lang.reflect.Method.invoke (Method.java:580)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
    at java.lang.reflect.Method.invoke (Method.java:580)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:39)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:122)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:61)
Caused by: org.apache.maven.plugin.MojoExecutionException: Failed to build quarkus application
    at io.quarkus.maven.BuildMojo.doExecute (BuildMojo.java:190)
    at io.quarkus.maven.QuarkusBootstrapMojo.execute (QuarkusBootstrapMojo.java:183)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:370)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:351)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:171)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:163)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:294)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:960)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
    at java.lang.reflect.Method.invoke (Method.java:580)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
    at java.lang.reflect.Method.invoke (Method.java:580)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:39)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:122)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:61)
Caused by: java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
	[error]: Build step io.quarkus.deployment.pkg.steps.NativeImageBuildStep#build threw an exception: io.quarkus.deployment.pkg.steps.NativeImageBuildStep$ImageGenerationFailureException: Image generation failed
	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.imageGenerationFailed(NativeImageBuildStep.java:616)
	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.build(NativeImageBuildStep.java:306)
	at java.base/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:733)
	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:811)
	at io.quarkus.builder.BuildContext.run(BuildContext.java:242)
	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2651)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2630)
	at org.jboss.threads.EnhancedQueueExecutor.runThreadBody(EnhancedQueueExecutor.java:1622)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1589)
	at java.base/java.lang.Thread.run(Thread.java:1583)
	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
Caused by: io.smallrye.common.process.AbnormalExitException: Process exited abnormally (pid 219) with exit code 1 with error output:
 > Fatal error: com.oracle.graal.pointsto.util.AnalysisError$ParsingError: Error encountered while parsing io.netty.handler.ssl.SslContext.newClientContextInternal(SslContext.java:844) 
 > Parsing context:
 >    at io.netty.handler.ssl.SslContextBuilder.build(SslContextBuilder.java:648)
 >    at io.grpc.netty.ProtocolNegotiators.from(ProtocolNegotiators.java:146)
 >    at io.grpc.netty.NettyChannelProvider.newChannelBuilder(NettyChannelProvider.java:50)
 > … (skipped 97 line(s)) …
 > 	at io.netty.internal.tcnative.SSLPrivateKeyMethod.<clinit>(SSLPrivateKeyMethod.java:22)
 > 	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized0(Native Method)
 > 	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized(Unsafe.java:1160)
 > 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.ClassInitializationSupport.ensureClassInitialized(ClassInitializationSupport.java:177)
 > 	... 75 more
	at io.smallrye.common.process.PipelineRunner.collectProblems(PipelineRunner.java:559)
	at io.smallrye.common.process.ProcessRunner.complete(ProcessRunner.java:165)
	at io.smallrye.common.process.ProcessRunner.run(ProcessRunner.java:95)
	at io.smallrye.common.process.ProcessBuilderImpl.run(ProcessBuilderImpl.java:199)
	at io.quarkus.deployment.pkg.steps.NativeImageBuildRunner.build(NativeImageBuildRunner.java:77)
	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.build(NativeImageBuildStep.java:300)
	... 10 more
    at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment (AugmentActionImpl.java:433)
    at io.quarkus.runner.bootstrap.AugmentActionImpl.createProductionApplication (AugmentActionImpl.java:186)
    at io.quarkus.maven.BuildMojo.doExecute (BuildMojo.java:143)
    at io.quarkus.maven.QuarkusBootstrapMojo.execute (QuarkusBootstrapMojo.java:183)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:370)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:351)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:171)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:163)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:294)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:960)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
    at java.lang.reflect.Method.invoke (Method.java:580)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
    at java.lang.reflect.Method.invoke (Method.java:580)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:39)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:122)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:61)
Caused by: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
	[error]: Build step io.quarkus.deployment.pkg.steps.NativeImageBuildStep#build threw an exception: io.quarkus.deployment.pkg.steps.NativeImageBuildStep$ImageGenerationFailureException: Image generation failed
	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.imageGenerationFailed(NativeImageBuildStep.java:616)
	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.build(NativeImageBuildStep.java:306)
	at java.base/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:733)
	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:811)
	at io.quarkus.builder.BuildContext.run(BuildContext.java:242)
	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2651)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2630)
	at org.jboss.threads.EnhancedQueueExecutor.runThreadBody(EnhancedQueueExecutor.java:1622)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1589)
	at java.base/java.lang.Thread.run(Thread.java:1583)
	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
Caused by: io.smallrye.common.process.AbnormalExitException: Process exited abnormally (pid 219) with exit code 1 with error output:
 > Fatal error: com.oracle.graal.pointsto.util.AnalysisError$ParsingError: Error encountered while parsing io.netty.handler.ssl.SslContext.newClientContextInternal(SslContext.java:844) 
 > Parsing context:
 >    at io.netty.handler.ssl.SslContextBuilder.build(SslContextBuilder.java:648)
 >    at io.grpc.netty.ProtocolNegotiators.from(ProtocolNegotiators.java:146)
 >    at io.grpc.netty.NettyChannelProvider.newChannelBuilder(NettyChannelProvider.java:50)
 > … (skipped 97 line(s)) …
 > 	at io.netty.internal.tcnative.SSLPrivateKeyMethod.<clinit>(SSLPrivateKeyMethod.java:22)
 > 	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized0(Native Method)
 > 	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized(Unsafe.java:1160)
 > 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.ClassInitializationSupport.ensureClassInitialized(ClassInitializationSupport.java:177)
 > 	... 75 more
	at io.smallrye.common.process.PipelineRunner.collectProblems(PipelineRunner.java:559)
	at io.smallrye.common.process.ProcessRunner.complete(ProcessRunner.java:165)
	at io.smallrye.common.process.ProcessRunner.run(ProcessRunner.java:95)
	at io.smallrye.common.process.ProcessBuilderImpl.run(ProcessBuilderImpl.java:199)
	at io.quarkus.deployment.pkg.steps.NativeImageBuildRunner.build(NativeImageBuildRunner.java:77)
	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.build(NativeImageBuildStep.java:300)
	... 10 more
    at io.quarkus.builder.Execution.run (Execution.java:139)
    at io.quarkus.builder.BuildExecutionBuilder.execute (BuildExecutionBuilder.java:76)
    at io.quarkus.deployment.QuarkusAugmentor.run (QuarkusAugmentor.java:182)
    at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment (AugmentActionImpl.java:429)
    at io.quarkus.runner.bootstrap.AugmentActionImpl.createProductionApplication (AugmentActionImpl.java:186)
    at io.quarkus.maven.BuildMojo.doExecute (BuildMojo.java:143)
    at io.quarkus.maven.QuarkusBootstrapMojo.execute (QuarkusBootstrapMojo.java:183)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:370)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:351)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:171)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:163)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:294)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:960)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
    at java.lang.reflect.Method.invoke (Method.java:580)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
    at java.lang.reflect.Method.invoke (Method.java:580)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:39)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:122)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:61)
Caused by: io.quarkus.deployment.pkg.steps.NativeImageBuildStep$ImageGenerationFailureException: Image generation failed
    at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.imageGenerationFailed (NativeImageBuildStep.java:616)
    at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.build (NativeImageBuildStep.java:306)
    at java.lang.invoke.MethodHandle.invokeWithArguments (MethodHandle.java:733)
    at io.quarkus.deployment.ExtensionLoader$3.execute (ExtensionLoader.java:811)
    at io.quarkus.builder.BuildContext.run (BuildContext.java:242)
    at org.jboss.threads.ContextHandler$1.runWith (ContextHandler.java:18)
    at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith (EnhancedQueueExecutor.java:2651)
    at org.jboss.threads.EnhancedQueueExecutor$Task.run (EnhancedQueueExecutor.java:2630)
    at org.jboss.threads.EnhancedQueueExecutor.runThreadBody (EnhancedQueueExecutor.java:1622)
    at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run (EnhancedQueueExecutor.java:1589)
    at java.lang.Thread.run (Thread.java:1583)
    at org.jboss.threads.JBossThread.run (JBossThread.java:501)
Caused by: io.smallrye.common.process.AbnormalExitException: Process exited abnormally (pid 219) with exit code 1 with error output:
 > Fatal error: com.oracle.graal.pointsto.util.AnalysisError$ParsingError: Error encountered while parsing io.netty.handler.ssl.SslContext.newClientContextInternal(SslContext.java:844) 
 > Parsing context:
 >    at io.netty.handler.ssl.SslContextBuilder.build(SslContextBuilder.java:648)
 >    at io.grpc.netty.ProtocolNegotiators.from(ProtocolNegotiators.java:146)
 >    at io.grpc.netty.NettyChannelProvider.newChannelBuilder(NettyChannelProvider.java:50)
 > … (skipped 97 line(s)) …
 > 	at io.netty.internal.tcnative.SSLPrivateKeyMethod.<clinit>(SSLPrivateKeyMethod.java:22)
 > 	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized0(Native Method)
 > 	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized(Unsafe.java:1160)
 > 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.classinitialization.ClassInitializationSupport.ensureClassInitialized(ClassInitializationSupport.java:177)
 > 	... 75 more
    at io.smallrye.common.process.PipelineRunner.collectProblems (PipelineRunner.java:559)
    at io.smallrye.common.process.ProcessRunner.complete (ProcessRunner.java:165)
    at io.smallrye.common.process.ProcessRunner.run (ProcessRunner.java:95)
    at io.smallrye.common.process.ProcessBuilderImpl.run (ProcessBuilderImpl.java:199)
    at io.quarkus.deployment.pkg.steps.NativeImageBuildRunner.build (NativeImageBuildRunner.java:77)
    at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.build (NativeImageBuildStep.java:300)
    at java.lang.invoke.MethodHandle.invokeWithArguments (MethodHandle.java:733)
    at io.quarkus.deployment.ExtensionLoader$3.execute (ExtensionLoader.java:811)
    at io.quarkus.builder.BuildContext.run (BuildContext.java:242)
    at org.jboss.threads.ContextHandler$1.runWith (ContextHandler.java:18)
    at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith (EnhancedQueueExecutor.java:2651)
    at org.jboss.threads.EnhancedQueueExecutor$Task.run (EnhancedQueueExecutor.java:2630)
    at org.jboss.threads.EnhancedQueueExecutor.runThreadBody (EnhancedQueueExecutor.java:1622)
    at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run (EnhancedQueueExecutor.java:1589)
    at java.lang.Thread.run (Thread.java:1583)
    at org.jboss.threads.JBossThread.run (JBossThread.java:501)
168235 [ERROR] 
168235 [ERROR] Re-run Maven using the -X switch to enable full debug logging.
168235 [ERROR] 
168235 [ERROR] For more information about the errors and possible solutions, please read the following articles:
168236 [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
Cleaning up project directory and file based variables
00:01
ERROR: Job failed: exit code 1
```

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### Mandrel or GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

--- a/extensions/netty/runtime/src/main/java/io/quarkus/netty/runtime/graal/NettySubstitutions.java
+++ b/extensions/netty/runtime/src/main/java/io/quarkus/netty/runtime/graal/NettySubstitutions.java
@@ -355,6 +355,20 @@ final class Target_io_netty_handler_ssl_SslContext {
             long sessionCacheSize, long sessionTimeout, boolean enableOcsp,
             SecureRandom secureRandom, String keyStoreType, String endpointIdentificationAlgorithm,
             Map.Entry<SslContextOption<?>, Object>... options) throws SSLException {
+        return newClientContextInternal(provider, sslContextProvider, trustCert, trustManagerFactory, keyCertChain, key,
+                keyPassword, keyManagerFactory, ciphers, cipherFilter, apn, protocols, sessionCacheSize, sessionTimeout, false,
+                enableOcsp, secureRandom, keyStoreType, endpointIdentificationAlgorithm, options);
+    }
+
+    @Substitute
+    static SslContext newClientContextInternal(SslProvider provider,
+            Provider sslContextProvider,
+            X509Certificate[] trustCert, TrustManagerFactory trustManagerFactory,
+            X509Certificate[] keyCertChain, PrivateKey key, String keyPassword, KeyManagerFactory keyManagerFactory,
+            Iterable<String> ciphers, CipherSuiteFilter cipherFilter, ApplicationProtocolConfig apn, String[] protocols,
+            long sessionCacheSize, long sessionTimeout, boolean startTls, boolean enableOcsp,
+            SecureRandom secureRandom, String keyStoreType, String endpointIdentificationAlgorithm,
+            Map.Entry<SslContextOption<?>, Object>... options) throws SSLException {
         if (enableOcsp) {
             throw new IllegalArgumentException("OCSP is not supported with this SslProvider: " + provider);
         }

